SwiftRustIntegrationTestRunner/SwiftRustIntegrationTestRunner/Result.swift

```diff
@@ -41,5 +43,7 @@ extension ResultTransparentStruct: Error {}
 extension SameEnum: @unchecked Sendable {}
 extension SameEnum: Error {}
 
+extension AsyncResultOkEnum: @unchecked Sendable {}
```

Comment:
Hmm we'll want to move away from these `@unchecked Sendable {}`s in favor of enabling `swift-bridge` to emit `Sendable` extension.

In the meantime, you can make this `@unchecked Sendable` safe by replacing the inner `String` with a primitive such as `u32`.

https://github.com/chinedufn/swift-bridge/blob/636fa2774870c052a18f06d98ca5f6966c09bdd6/crates/swift-integration-tests/src/async_function.rs#L50-L54

Let's also leave a TODO in this file to ditch `@unchecked Sendable` in favor of proper `Sendable` support. Can link to #269

Reply:
For future reference, I think you linked the wrong issue by accident - the relevant one is #150.